backup-stores/s3/src/main/java/io/camunda/zeebe/backup/s3/S3Config.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.s3;
+
+public record S3Config(String bucketName) {}
```

Comment:
:wrench: Add documentation. Is this config related to backup only? Then I suggest it to rename accordingly - may be `S3BackupConfig`.

Reply:
:+1: Adjusted the name, but I kept the documentation very light for now.